deps/exokit-bindings/browser/src/browser.cpp

```diff
@@ -578,18 +627,25 @@ NAN_METHOD(Browser::SendKeyUp) {
 }
 
 NAN_METHOD(Browser::SendKeyPress) {
-  if (info[0]->IsNumber()) {
+  // Nan::HandleScope scope;
+  if (info[0]->IsNumber() && info[1]->IsObject()) {
     Browser *browser = ObjectWrap::Unwrap<Browser>(info.This());
     int key = info[0]->Uint32Value();
+	
+	Local<Object> modifiersObj = Local<Object>::Cast(info[1]);
+	int modifiers = GetKeyModifiers(modifiersObj);
+	if(modifiersObj->Get(JS_STR("shiftKey"))->BooleanValue()){
```

Comment:
This is the current bug: CEF not handling shift.

Reply:
this does hit this block ok, so this part seems right